l10n_es_aeat/views/account_journal_view.xml

```diff
@@ -6,8 +6,17 @@
         <field name="model">account.journal</field>
         <field name="inherit_id" ref="account.view_account_journal_form" />
         <field name="arch" type="xml">
-            <xpath expr="//field[@name='type'][last()]" position="after">
-                <field name="thirdparty_invoice" invisible="1" />
+            <xpath expr="//page[@name='advanced_settings']" position="before">
+                <page name="l10n_es_aeat" string="Localization Settings">
```

Comment:
shouldn't this be filtered just for sale and purchase questions?

Reply:
Y solo visible si el país de la compañía es España.